src/promptflow/tests/executor/unittests/_core/test_tool_resolver.py

```diff
@@ -0,0 +1,373 @@
+from pathlib import Path
```

Comment:
We should put this file in unittests/executor folder since _tool_resolver.py is located in executor/ folder.

Reply:
Moved to unittests/executor folder.